cql3/statements/alter_keyspace_statement.cc

```diff
@@ -50,6 +50,13 @@ void cql3::statements::alter_keyspace_statement::validate(query_processor& qp, c
         }
         try {
             auto ks = qp.db().find_keyspace(_name);
+            // TODO: the full tablets support will be implemented in https://github.com/scylladb/scylladb/issues/16129
+            if (ks.get_replication_strategy().uses_tablets() && !_attrs->get_initial_tablets(ks.metadata()->strategy_name(), true).has_value()) {
+                throw exceptions::invalid_request_exception("Disabling tablets via ALTER KEYSPACE statement is not yet supported for keyspaces with tablets.");
```

Comment:
The comment above suggest that this limitation is temporary, until https://github.com/scylladb/scylladb/issues/16129, but didn't we say we'll never support **disabling** tablets? And isn't this unrelated to #16129?

Reply:
Resolved: https://github.com/scylladb/scylladb/pull/18772/files#r1617888067

Reply:
OK, removed.